source/plugins/entities.rst

```diff
@@ -2,3 +2,33 @@
 Working with Entities
 =====================
 
+Entities are a huge part of Minecraft, and SpongeAPI in general. All entities exist within worlds
+
+Spawning an Entity
+==================
+
+In almost all cases, to spawn an entity, you need an `EntityType <https://github.com/SpongePowered/SpongeAPI/blob/master/src/main/java/org/spongepowered/api/entity/EntityType.java>`__, a `World <https://github.com/SpongePowered/SpongeAPI/blob/master/src/main/java/org/spongepowered/api/world/World.java>`__, and a `Location <https://github.com/SpongePowered/SpongeAPI/blob/master/src/main/java/org/spongepowered/api/world/Location.java>`__ to do so.
+
```

Comment:
spongepowered.github.io/SpongeAPI/ There's another page that's using the jdocs too